webapp/portlet/src/main/webapp/vue-apps/attach-image/components/AttachedItems.vue

```diff
@@ -39,6 +47,16 @@ export default {
     attachmentsCount() {
       return this.attachments.length;
     },
+  },
+  methods: {
+    openPreview(attachmentId) {
+      this.previewAttachments = this.activity?.metadatas?.attachments?.map(metadata => ({
+        id: metadata.name,
+        filename: metadata.properties.fileName,
```

Comment:
can you please add the other properties here to help on future evolution

Reply:
added